assets/src/edit-story/app/media/useUploadMedia/useUploadMedia.js

```diff
@@ -56,28 +48,13 @@ function useUploadMedia({ media, setMedia }) {
       if (!files || files.length === 0) {
         return;
       }
+
       let localFiles;
       let updatedMedia;
       try {
         setIsUploading(true);
         setPreventUnload('upload', true);
 
-        files.reverse().map((file) => {
-          if (!isValidType(file)) {
-            /* translators: %s is a list of allowed file extensions. */
-            const message = sprintf(
-              /* translators: %s: list of allowed file types. */
-              __('Please choose only %s to upload.', 'web-stories'),
-              allowedFileTypes.join(
-                /* translators: delimiter used in a list */
-                __(', ', 'web-stories')
-              )
-            );
-
-            throw createError('ValidError', file.name, message);
-          }
-        });
```

Comment:
Why is this being removed? Is this handled somewhere else now?

Reply:
Yes, in `useUploader`.

This is to allow uploading MOV files because we can transcode them.

Otherwise MOV files will be blocked.

Reply:
What if core doesn't support the file type?

Reply:
Not sure I'm following.

There is still a check for allowed file types, just in a different place.
This way we can transcode MOV files if uploaded.

But if you upload another file type that is not supported, it still fails.

Reply:
What is there is video file that core doesn't support, like a file with mimetype video/wibble.

Reply:
If it's a video format supported by ffmpeg it will be transcoded.

If it isn't, it's rejected.

This will be handled by #6012